Services/CartService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using ConsoleApp2.Models;
+using ConsoleApp2.Repositories;
+
+namespace ConsoleApp2.Services
+{
+
+    
+        public class CartService
+        {
+            private readonly CartRepository _cartRepository;
+            public CartService(CartRepository cartRepository)
+            {
+                _cartRepository = cartRepository;
+            }
+
```

Comment:
Where is the code that confirms a purchase? "Clean cart, and remove purchased items from the product repository"?